src/Abstractions/NexusMods.Abstractions.FileStore/Trees/ModFileTree.cs

```diff
@@ -170,7 +170,7 @@ public static class ModFileTreeExtensions
     /// </summary>
     /// <param name="keyedBox">The KeyedBox containing a ModFileTree instance.</param>
     /// <returns>The file name.</returns>
-    public static RelativePath FileName(this ModFileTreeNode keyedBox) => keyedBox.Item.FileName;
+    public static RelativePath FileName(this ModFileTreeNode keyedBox) => keyedBox.Item.Path;
```

Comment:
I think this was unintended?

Reply:
Not sure, I'm going to gut it though as all the `FileName` `Path` stuff is making it hard to know what's going on. I'd rather this code work against the LoadoutItems directly

Reply:
Oh wait, in that file yeah that was a mistake let me back that out